src/analog/commands/downlink/setParameter.ts

```diff
@@ -530,6 +530,37 @@ export const examples: command.TCommandExamples = {
             0x03, 0x02,
             0x3b, 0x01
         ]
+    },
+    'set nbiot ntp server': {
+        id,
+        name,
+        headerSize,
+        parameters: {
+            id: 60,
```

Comment:
how about switch to constant usage (this place and above)?

```suggestion
            id: deviceParameters.NBIOT_NTP_SERVER,
            name: deviceParameterNames.NBIOT_NTP_SERVER,
```

to be sure there are no mistypes

Reply:
fixed. The example names are also changed to make it clear what you are selecting when creating a message in the Jooby GUI.